### ISSUE TITLE
As a data Browser user I want to use filters and search also for the mobility API, to search and filter for all data sets

### ISSUE THREAD

**Issue by sseppi** [noi-techpark/it.bz.opendatahub.databrowser#427]

**Comment by mrabans** [comment]
This issue will be initiated after https://github.com/noi-techpark/it.bz.opendatahub.databrowser/issues/296

**Comment by gappc** [comment]
@sseppi sort seems to not be supported by the Mobility API at the moment, I deactivated the feature for the Mobility API in the Data Browser. Is this ok?

**Comment by sseppi** [comment]
@gappc yes, for the moment we need to focus on showing the data of the Mobility API. In a second round we will focus on oprimization

**Comment by gappc** [comment]
@sseppi since search and filter are implemented and the mobility API does not support sorting, can we close this issue?

**Comment by sseppi** [comment]
@gappc I created a dedicated issue for the sorting, since I think that is a useful feature for the Data Browser user. In the issue #517 I mentioned also the fact that we need to extend the API.
